### PR TITLE
Wait for {{ jenkins_port }} to become available

### PR DESCRIPTION
**Pull request by wzur**
Added waiting for a port Jenkins binds to, to avoid problems with undefined jenkins_login_page when connection to that port is refused

### CODE REVIEW

**Comment by karlmdavis** [comment]
This is an intermittent problem. That said, I did see it happen on a run I made this past Thursday.

Since this PR now conflicts with `master`, I'm going to fix it up and get it merged (if I can figure out how to do that, otherwise I'll close it and open a new one). Thanks very much for the contribution!

**Comment by karlmdavis** [comment]
Sign. Looks like there's no way for me to amend this PR and preserve you in the history, which sucks. To try & avoid doing that to you, I'll let it sit for a bit (until I swing by this project again). If you can fix up the PR by then, I'll happily merge it. Otherwise, I'll close it and file a new one.

Again: thanks for the contribution!

File: handlers/restart_service.yml [hunk 1]
```diff
@@ -6,6 +6,9 @@
     state: restarted
   become: true
 
+- name: Wait for Jenkins to bind to TCP port before checking HTTP
```
Comment (karlmdavis) [inline handlers/restart_service.yml]: Just to keep things succinct, could you please rename this to "Wait for Jenkins to Open Port"? Thanks!

**Comment by wzur** [comment]
Hi Karl,

I'm replacing this PR with PR #8 

Waldemar

**Pull request closed without merging**